### ISSUE TITLE
Acekard Theme bug with Cheats

### ISSUE THREAD

**Issue by Pixsky8** [DS-Homebrew/TWiLightMenu#696]
Im having issue with the Acekard theme. 

Whenever i change the cheat codes i use with this theme (both the 3ds and dsi themes work fine, i did not test the R4 and the Staturn theme) the game does not boot up and stays on a white screen.
Restarting Twilight menu does not fix the problem and it affects every game afterwards.

The only fix i found was to delete the _nds folder and recopy it.

Im using Twilight menu 9.0.0 with nds-bootstrap Release v0.22.0 on a O3DS with luma 9.1-89cc6ed.

Im sorry if i forgot to mention something and for the formatting, this is my first time posting an issue on github.

**Comment by RocketRobz** [comment]
Did you try turning off the cheats?

**Comment by NightYoshi370** [comment]
Can you post an example of the cheat file when created in DSi/3DS theme compared to when created in the Acekard theme?

**Comment by Pixsky8** [comment]
I've replaced my _nds folder with the one from the new update and it is working properly now. I probably did something wrong when i updated it or messed up somewhere in the settings.

I also tried to take my _nds before the update and update it and it worked too. I probably hit the wrong button when windows asked me to replace the files.


> Did you try turning off the cheats?

I tried with the old _nds but it didn't work.


> Can you post an example of the cheat file when created in DSi/3DS theme compared to when created in the Acekard theme?

I don't have it anymore. I didn't expected that anyone would reply me that fast.

I'm sorry to have bothered you and many thanks for the quick replies.
